src/Symfony/Bridge/Monolog/Processor/SwitchUserTokenProcessor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Processor;
+
+use Symfony\Component\Security\Core\Authentication\Token\SwitchUserToken;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+/**
+ * Adds the original security token to the log entry.
+ *
+ * @author Igor Timoshenko <igor.timoshenko@i.ua>
+ */
+class SwitchUserTokenProcessor extends AbstractTokenProcessor
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function getKey(): string
+    {
+        return 'original_token';
```

Comment:
I know it's the legacy naming, but I don't really like `original_token` which kind of miss some context. What about `impersonating_token` or `impersonator_token` instead?

Reply:
I have used the `original_token` key because of the original (impersonator) token's getter has name `getOriginalToken`. As for me, the `impersonator_token` key is more suitable, but I am not sure that is a good idea to change the `original_token` key to `impersonator_token` until the original token's getter is renamed to `getImpersonatorToken`

Reply:
As someone looking at the logs or Sentry reports for instance, `impersonator_token` is more meaningful than `original_token` though. The internal name in the code does not matter in this regard.

Reply:
I can rename

Reply:
I would use `impersonator_token`.

Reply:
Changed to `impersonator_token`